src/software/logger/logger.h

```diff
@@ -1,5 +1,11 @@
 #pragma once
 
+#if __cplusplus > 201703L
+#include <filesystem>
+#else
+#include <experimental/filesystem>
+#endif
+
```

Comment:
I think we compile for c++ 20 through bazel rules so this isn't necessary I think

Reply:
apparently we need the experimental, it doesnt work otherwise